File: src/main/java/sk/hudak/prco/dto/product/ProductUnitDataDto.java
package sk.hudak.prco.dto.product;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import sk.hudak.prco.dto.DtoAble;

import java.math.BigDecimal;

@Getter
@Setter
@ToString
public class ProductUnitDataDto implements DtoAble {
    // read only
    private Long id;
    private String name;

    // edit:
    private String unit;
    private BigDecimal unitValue;
    private Integer unitPackageCount;
}

File: src/main/java/sk/hudak/prco/dto/product/ProductUnitDataDto.kt
package sk.hudak.prco.dto.product

import sk.hudak.prco.dto.DtoAble
import java.math.BigDecimal

data class ProductUnitDataDto(
        // read only
        var id: Long? = null,
        var name: String? = null,
        // edit:
        var unit: String? = null,
        var unitValue: BigDecimal? = null,
        var unitPackageCount: Int? = null) : DtoAble
